### PR TITLE
Simple Codebase Improvement  

### PR DESCRIPTION
**Pull request by Jhooomn**
feat: it was added a string variable to avoid replication one.

### CODE REVIEW

File: app/src/androidTest/java/org/schabi/newpipe/local/playlist/LocalPlaylistManagerTest.kt [hunk 1]
```diff
@@ -43,10 +43,11 @@ class LocalPlaylistManagerTest {
 
     @Test
     fun createPlaylist() {
+        val NEW_PIPE_LINK = "https://newpipe.net/"
```
Comment (Stypox) [inline app/src/androidTest/java/org/schabi/newpipe/local/playlist/LocalPlaylistManagerTest.kt:46]: Call it `NEWPIPE_URL`

**Review by Stypox — CHANGES_REQUESTED** [review]
Thank you!

File: app/src/androidTest/java/org/schabi/newpipe/local/playlist/LocalPlaylistManagerTest.kt (hunk 1 above)
Comment (Jhooomn) [reply app/src/androidTest/java/org/schabi/newpipe/local/playlist/LocalPlaylistManagerTest.kt]: It was updated with recommendations.

**Review by mhmdanas — DISMISSED** [review]
Looks good to me, but a couple things:
- please use more descriptive commit messages. I know that the message you used is the one provided by GitHub by default, but you can change it.
- please don't remove the template that was there next time you open a PR (or issue for that matter).

Thanks!

**Review by Stypox — APPROVED** [review]

**Review by mhmdanas — APPROVED** [review]

**Pull request merged**